spp_ethnic_group/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Part of OpenSPP. See LICENSE file for full copyright and licensing details.
+
+
+{
+    "name": "OpenSPP: Ethnic Group",
+    "category": "OpenSPP",
+    "version": "17.0.1.0.0",
+    "sequence": 1,
+    "author": "OpenSPP.org",
+    "website": "https://github.com/OpenSPP/openspp-modules",
+    "license": "LGPL-3",
+    "development_status": "Beta",
```

Comment:
Is this development status correct?

Reply:
@kneckinator I changed it to Alpha since the index.html is in Alpha.

Reply:
I think (@jeremi or @dasunhegoda to confirm) that the change should go in the other direction, changing it to alpha in `index.html`. My reasoning is that all other OpenSPP modules are either in Beta or higher.

Reply:
@kneckinator @reichie020212 : let's make it `beta` to be consistant.

Reply:
I also agree that it should be beta. For future escalation of development_status, how do we determine what should be set in a module?

Reply:
@reichie020212 FYI/A

Reply:
@reichie020212 Could you please set the version to `beta` in the relevant files? Thanks.

Reply:
Noted on this @kneckinator

Reply:
@kneckinator done changing from Alpha to Beta

Reply:
Awesome, thanks @reichie020212